pyvista/plotting/widgets.py

```diff
@@ -81,6 +81,18 @@ def add_box_widget(
             If ``True``, the widget will be passed as the last
             argument of the callback.
 
+        Examples
```

Comment:
Examples needs to be after Returns, and there are two Examples sections now.

Reply:
Fixed in 762441f709ac2f8950441c33a2883855543824af.